CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+## v0.9.0 [2021-11-23]
+
+_What's new?_
+
+- New tables added
+  - [github_my_star](https://hub.steampipe.io/plugins/turbot/github/tables/github_my_star) ([#90](https://github.com/turbot/steampipe-plugin-github/pull/90))
+
+_Enhancements_
+
+- Recompiled plugin with [steampipe-plugin-sdk v1.8.2](https://github.com/turbot/steampipe-plugin-sdk/blob/main/CHANGELOG.md#v182--2021-11-22) ([#95](https://github.com/turbot/steampipe-plugin-github/pull/95))
+- Update all the tables which uses custom retry logic to use RetryHydrate function from Steampipe plugin SDK ([#86](https://github.com/turbot/steampipe-plugin-github/pull/86))
+- Add additional optional key quals, filter support, and context cancellation handling across the following tables ([#61](https://github.com/turbot/steampipe-plugin-github/pull/61))
+  - `github_branch`
+  - `github_commit`
+  - `github_issue`
+  - `github_my_gist`
+  - `github_my_issue`
+  - `github_my_organization`
+  - `github_my_repository`
+  - `github_my_team`
+  - `github_pull_request`
+  - `github_release`
+  - `github_stargazers`
+  - `github_tag`
+  - `github_workflow`
+  - `github_branch`
+  - `github_commit`
+  - `github_issue`
+  - `github_my_issue`
+  - `github_my_repository`
+  - `github_pull_request`
+
+_Bug fixes_
+
+- `github_repository` table will now return an empty row instead of not found error when the repository collaborator details are not available ([#89](https://github.com/turbot/steampipe-plugin-github/pull/89))
```

Comment:
```suggestion
- `github_repository` table will now return an empty row instead of `not found` error when the repository collaborator details are not available ([#89](https://github.com/turbot/steampipe-plugin-github/pull/89))
```